sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -304,11 +304,43 @@ export interface SignedIdentifier {
     id: string;
 }
 
+// @public
+export interface TableBatch {
+    createEntities: <T extends object>(entitites: TableEntity<T>[]) => void;
+    createEntity: <T extends object>(entity: TableEntity<T>) => void;
+    deleteEntity: (partitionKey: string, rowKey: string, options?: DeleteTableEntityOptions) => void;
+    partitionKey: string;
+    submitBatch: () => Promise<TableBatchResponse>;
+    updateEntity: <T extends object>(entity: TableEntity<T>, mode: UpdateMode, options?: UpdateTableEntityOptions) => void;
+}
+
+// @public
+export interface TableBatchEntityResponse {
+    body?: any;
+    etag?: string;
+    rowKey?: string;
+    status: number;
+}
```

Comment:
should this be of type `T` so you can type the response?

```suggestion
export interface TableBatchEntityResponse<T> {
    body?: T;
    etag?: string;
    rowKey?: string;
    status: number;
}
```

Reply:
Thanks for bringing this up! the current implementation will never return content in the sub-requests unless it is an error. So I'm removing body throwing the error instead

If we decide to support query in the batch, it is possible that we'll want to support getting a body. At that time we should discuss how to type the bodies, as each sub-request may have different shapes